### PR TITLE
Blog on SIG Node spotlight 2021

### PR DESCRIPTION
**Pull request by dewan-ahmed**
This is a blog on SIG Node spotlight capturing my conversations with [Elana Hashman](https://twitter.com/ehashdn) & [Sergey Kanzhelev](https://twitter.com/SergeyKanzhelev).

### CODE REVIEW

File: content/en/blog/_posts/2021-08-13-SIG-Node-Spotlight/index.md
Comment (reylejano) [inline content/en/blog/_posts/2021-08-13-SIG-Node-Spotlight/index.md:58]:
```suggestion
SK: SIG Node works on many workstreams in very different areas. All of these areas are on system level. For the typical code contributions you need to have a passion for building and utilizing low level APIs and writing performant and reliable components. Being a contributor you will learn how to debug and troubleshoot, profile, and monitor these components, as well as user workload that is run by these components. Often, with the limited to no access to Nodes, as they are running production workloads.
```
nit: extra space

File: content/en/blog/_posts/2021-08-13-SIG-Node-Spotlight/index.md
Comment (reylejano) [inline content/en/blog/_posts/2021-08-13-SIG-Node-Spotlight/index.md:55]:
```suggestion
```

File: content/en/blog/_posts/2021-08-13-SIG-Node-Spotlight/index.md
Comment (reylejano) [inline content/en/blog/_posts/2021-08-13-SIG-Node-Spotlight/index.md:4]:
```suggestion
date: 2021-09-06
```
Next available publish date is Sept 6

File: content/en/blog/_posts/2021-08-13-SIG-Node-Spotlight/index.md
Comment (reylejano) [inline content/en/blog/_posts/2021-08-13-SIG-Node-Spotlight/index.md:14]:
```suggestion
```
nit: remove extra line break

File: content/en/blog/_posts/2021-08-13-SIG-Node-Spotlight/index.md
Comment (reylejano) [inline content/en/blog/_posts/2021-08-13-SIG-Node-Spotlight/index.md:21]:
```suggestion
```
nit: remove extra line breaks

File: content/en/blog/_posts/2021-08-13-SIG-Node-Spotlight/index.md
Comment (reylejano) [inline content/en/blog/_posts/2021-08-13-SIG-Node-Spotlight/index.md:22]:
```suggestion

```
nit: remove extra line breaks

Comment (reylejano) [inline content/en/blog/_posts/2021-08-13-SIG-Node-Spotlight/index.md:22]:
```suggestion
```
nit: remove extra line breaks

**Comment by reylejano** [comment]
Hi @ehashman and @SergeyKanzhelev ,

Please review this blog PR as it's an interview with you about SIG Node. 
Please add comments or suggestions if needed or give an `/lgtm` if this interview is ok and ready to publish as a blog.

Thank you!

**Comment by ehashman** [comment]
/assign @ehashman @SergeyKanzhelev 

suspect it will help if we actually get tagged on GitHub, I forgot about this :sweat_smile:

**Review by SergeyKanzhelev — APPROVED** [review]
/lgtm

thanks!

**Review by ehashman — COMMENTED** [review]
Checked over my answers, they look good -- will let Sergey sign off on his.

Do we want to add links?

**Comment by sftim** [comment]
Blog team, is this looking good to merge? If it is, let's pick a publication date (we can push an extra commit to set that).

**Comment by nate-double-u** [comment]
I'd suggest that we wait for the author to change up the introduction before we merge.

**Comment by reylejano** [comment]
The author, @dewan-ahmed , is out until Sept 17. 
@dewan-ahmed please take a look at @nate-double-u's [comment](https://github.com/kubernetes/website/pull/29380/files#r696125785) and @ehashman's [comment](https://github.com/kubernetes/website/pull/29380#discussion_r704661435) when you're back.

**Comment by dewan-ahmed** [comment]
Thanks for the note @reylejano 

I'll address the above comments via another commit by EOD on Sept22.

**Review by sftim — COMMENTED** [review]
Publication date is now in the past
/hold

as we need to reschedule (SIG Docs blog team can take care of this)

**Comment by dewan-ahmed** [comment]
Hi @reylejano @sftim I addressed @nate-double-u's [comment](https://github.com/kubernetes/website/pull/29380/files#r696125785) and @ehashman's [comment](https://github.com/kubernetes/website/pull/29380#discussion_r704661435). I also adjusted the date to Sept27, 2021 as a potential publication date.

Please let me know if you see any other fix required.

**Comment by reylejano** [comment]
Thank you for the updates
Re-applying lgtm
/lgtm

**Comment by reylejano** [comment]
Preview of blog: https://deploy-preview-29380--kubernetes-io-main-staging.netlify.app/blog/2021/09/27/sig-node-spotlight-2021/

@dewan-ahmed thank you for addressing the comments
/approve
/hold cancel

File: content/en/blog/_posts/2021-08-13-SIG-Node-Spotlight/index.md
Comment (nate-double-u) [inline content/en/blog/_posts/2021-08-13-SIG-Node-Spotlight/index.md:12]: This is a better introduction, but now it's mostly a copy of the answer to the first question so it's giving away some of the post's content early.

Since this PR has been merged, but the publication date is still in the future, I suggest opening a follow up PR to rewrite this introduction.

**Review by nate-double-u — COMMENTED** [review]
Thanks for updating the introduction, but I've still got some comments about it:

File: content/en/blog/_posts/2021-08-13-SIG-Node-Spotlight/index.md
Comment (dewan-ahmed) [reply content/en/blog/_posts/2021-08-13-SIG-Node-Spotlight/index.md:12]: Good catch @nate-double-u 

I didn't notice that the same was reused in the first question. I'm working on rewriting the intro and expect to create another PR by end of the day.

Comment (dewan-ahmed) [reply content/en/blog/_posts/2021-08-13-SIG-Node-Spotlight/index.md:12]: @nate-double-u https://github.com/kubernetes/website/pull/29787

**Pull request merged**
